CONTRIBUTING.md

```diff
@@ -119,4 +119,4 @@ blockName__elementName_modifierName_modifierValue
 
 ## Financial contributions
 
-We also welcome financial contributions in full transparency on our [open collective](https://opencollective.com/osca). Anyone can make a donation or file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
\ No newline at end of file
+We also welcome financial contributions in full transparency on our [open collective](https://opencollective.com/osca). Anyone can make a donation or file an expense. If the expense makes sense for the development of the community, it will be "merged" in the ledger of our open collective by the core contributors and the person who filed the expense will be reimbursed.
```

Comment:
Is there a reason why you made a change here?

Reply:
I am not sure why that is because I did not make any direct change in that file